src/viam/app/app_client.py

```diff
@@ -645,8 +912,7 @@ async def new_robot(self, name: str, location_id: Optional[str] = None) -> str:
         Returns:
             str: The new robot's ID.
         """
-        location_id = location_id if location_id else self._location_id
-        request = NewRobotRequest(name=name, location=location_id if location_id else "")
+        request = NewRobotRequest(location=location_id if location_id else self._location_id if self._location_id else "", name=name)
```

Comment:
(q) I see we combined the "define `location_id`" line and the `create a request` line into one in a bunch of places in this PR. just curious what the motivation for that was?

Reply:
No specific reason I think I did it just to compact everything down. Can def revert back to two lines for readability

Reply:
I personally prefer the two lines, but not enough to bother you to change back :) totally fine to leave as is.